app/renderer/components/Inspector/ElementLocator.js

```diff
@@ -33,14 +32,8 @@ const locatorStrategies = (driver) => {
   return strategies;
 };
 
-const showMissingAutomationNameInfo = (driver, t) => {
-  if (!driver.client.capabilities.automationName) {
-    return <Alert message={t('missingAutomationNameForStrategies')} type={ALERT.INFO} showIcon/>;
-  }
-};
-
 const ElementLocator = (props) => {
-  const { setLocatorTestValue, locatorTestValue, setLocatorTestStrategy, locatorTestStrategy, driver, t } = props;
+  const { setLocatorTestValue, locatorTestValue, setLocatorTestStrategy, locatorTestStrategy, automationName, t } = props;
```

Comment:
maybe it would make sence to integrate [prettify](https://www.npmjs.com/package/prettify) later.